src/data_distribution/base_distribution.hpp

```diff
@@ -81,6 +93,19 @@ namespace argo {
 					return 0;
 				}
 
+				/**
+				 * @brief compute offset into the home node's share of the memory
+				 * @param ptr address to find offset of
```

Comment:
Something is not OK in this sentence.

Reply:
This is reworded in the latest commit, let me know if the new version is okay.

Reply:
assuming kostis meant the parameter `ptr`, which is the address of which the offset is to be found, I think the comment is perfectly OK :)